creator-node/src/logging.js

```diff
@@ -130,11 +130,15 @@ function getDuration({ startTime }) {
  * @param {number} startTime the start time
  * @param {string} msg the message to print
  */
-function logInfoWithDuration({ logger, startTime }, msg) {
+function logInfoWithDuration(
+  { logger, startTime },
+  msg,
+  durationKey = 'duration'
+) {
   const durationMs = getDuration({ startTime })
 
   if (durationMs) {
-    logger.info({ duration: durationMs }, msg)
+    logger.info({ [durationKey]: durationMs }, msg)
```

Comment:
these changes were all part of an earlier PR so not sure why they're showing up here, but it's just adding durations to logs and was tested